tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -152,3 +158,30 @@ def test_arxiv_derive_inspire_categories_does_nothing_with_existing_categories()
 
     assert validate(result, inspire_categories_schema) is None
     assert expected == result
+
+
+@patch('inspirehep.modules.workflows.tasks.arxiv.os')
+def test_arxiv_author_list_handles_auto_ignore_from_arxiv(mock_os):
+    data = {
+        'arxiv_eprints': {
+            'value': ['1703.09986']
+        }
+    }
```

Comment:
Note that this is incorrect: arxiv_eprints should be a list, not an object. This is why I started peppering our tests with assertions about the schema everytime we write code that relies on it. For example, a few lines above: https://github.com/inspirehep/inspire-next/blob/c14f5327de85f56da9a6565997a10fd6b3c71c43/tests/unit/workflows/test_workflows_tasks_arxiv.py#L137-L138